src/pages/Login/Login.js

```diff
@@ -1,8 +1,131 @@
 import React, { Component } from 'react';
+import IdPwInput from './IdPwInput';
+import './Login.scss';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btn: false,
+    };
+  }
+
+  handleInputs = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleLogin = () => {
+    console.log('click');
+    fetch('http://10.58.4.133:8000/user/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(result => result.json())
+      .then(result => {
+        if (result.MESSAGE === 'SUCCESS') {
+          alert(`Welcome ${this.state.id}`);
+          localStorage.setItem('dot-token', result.token);
+          this.props.history.push('/');
+        } else {
+          alert('Login failed');
+        }
+      });
+  };
+
   render() {
-    return <div></div>;
+    const IDPW = [
+      {
+        name: 'id',
+        title: '아이디',
+        type: 'text',
+        onchanging: this.handleInputs,
+      },
+      {
+        name: 'pw',
+        title: '비밀번호',
+        type: 'password',
+        onchanging: this.handleInputs,
+      },
+    ];
+
+    const checkIdPw =
+      this.state.id.includes('@') && this.state.pw.length > 7 ? true : false;
+    console.log(this.state);
+    return (
+      <div className="wrapLogin">
+        <header className="header">
+          <div className="headerInner">
+            <a href="/" className="logo">
+              <h1 className="logoText">Dot Friends</h1>
+            </a>
+          </div>
+        </header>
+
+        {/* content */}
+        <div className="content">
+          <div className="loginWrap">
+            <div className="idPwWrap">
+              {IDPW.map((input, idx) => {
+                return (
+                  <IdPwInput
+                    key={idx}
+                    name={input.name}
+                    title={input.title}
+                    type={input.type}
+                    onchanging={input.onchanging}
+                    loginkey={this.loginkey}
+                  />
```

Comment:
```suggestion
                  <IdPwInput
                    key={idx}
                    input={input}
                    onchanging={this.handleInputs}
                  />
```
1. loginkey 는 현재 사용하지 않으므로 삭제해주세요!
2. inpur.name, title, type을 따로 넘겨주기 보단 한 번에 보내는 것이 더 효율적입니다.
3. onChange 값에는 handleInput 함수를 직접 넣어주시는 편이 더 좋습니다!